src/inc/clrnt.h

```diff
@@ -997,6 +1010,19 @@ RtlVirtualUnwind (
     __out PDWORD EstablisherFrame,
     __inout_opt PT_KNONVOLATILE_CONTEXT_POINTERS ContextPointers
     );
+#else
```

Comment:
It seems that NTAPI is defined in the palrt.h, but you are right that the NTSYSAPI is undefined. Can you please just define NTSYSAPI in the palrt.h next to the NTAPI as empty and remove the `#ifdefs` here?